frontend/src/i18n/locales/en-US.json

```diff
@@ -168,24 +167,24 @@
     "audit-answer-contract": "The {contracts} have been audited by ConsenSys Diligence, and the audit report can be found {here}.",
     "audit-contracts": "contracts",
     "audit-here": "here",
-    "audit-answer-umrba-js": "The {umbra-js} {library} responsible for handling the required off-chain logic and elliptic curve operations—has been audited by Least Authority, and the audit report can be found {here}.",
+    "audit-answer-umrba-js": "The {umbrajs} {library} responsible for handling the required off-chain logic and elliptic curve operations—has been audited by Least Authority, and the audit report can be found {here}.",
```

Comment:
Probably a few other places to fix this typo too
```suggestion
    "audit-answer-umbra-js": "The {umbrajs} {library} responsible for handling the required off-chain logic and elliptic curve operations—has been audited by Least Authority, and the audit report can be found {here}.",
```

Reply:
From testing it seems like dashes aren't allowed in key's contents like the 

> "The {umbrajs} {library} responsible...

But dashes in key names are still allowed.

> "audit-answer-umbra-js"

So left the rest as is.

Reply:
Yea that's fine! The suggestion is that they key name is "audit-answer-**umrba**-js" instead of "audit-answer-**umbra**-js" 😅

Reply:
oh shoot, totally missed that. My bad.

Reply:
Fixed it. Thank you!

Reply:
Thanks!